src/test/java/seedu/address/logic/commands/VolunteerListEventCommandTest.java

```diff
@@ -0,0 +1,86 @@
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import static seedu.address.testutil.Assert.assertThrows;
+import static seedu.address.testutil.TypicalEvents.getTypicalEventStorage;
+import static seedu.address.testutil.TypicalIndexes.INDEX_FIRST;
+import static seedu.address.testutil.TypicalIndexes.INDEX_SECOND;
+import static seedu.address.testutil.TypicalVolunteers.getTypicalVolunteerStorage;
+
+import org.junit.jupiter.api.Test;
+
+import seedu.address.commons.core.index.Index;
+import seedu.address.logic.Messages;
+import seedu.address.logic.commands.eventvolunteercommands.VolunteerListEventCommand;
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+import seedu.address.model.ModelManager;
+import seedu.address.model.UserPrefs;
+import seedu.address.model.volunteer.Volunteer;
+
+public class VolunteerListEventCommandTest {
+    private Model model = new ModelManager(getTypicalEventStorage(), getTypicalVolunteerStorage(), new UserPrefs());
+
+    @Test
+    public void execute_invalidIndex_throwsCommandException() {
+        // Assign a volunteer to an event in the model
+        model.getEventStorage().getEventList().get(model.getFilteredEventList().size() - 1).addVolunteer(
+            model.getVolunteerStorage().getVolunteerList().get(model.getVolunteerStorage()
+                    .getVolunteerList().size() - 1));
+
+        Index invalidIndex = Index.fromOneBased(model.getFilteredVolunteerList().size() + 1);
+        VolunteerListEventCommand command = new VolunteerListEventCommand(invalidIndex);
+        assertThrows(CommandException.class, Messages.MESSAGE_INVALID_EVENT_DISPLAYED_INDEX, ()
+                -> command.execute(model));
+    }
+    @Test
+    public void execute_validIndexes_listSuccessful() {
+        // Assign a volunteer to an event in the model
+        model.getEventStorage().getEventList().get(model.getFilteredEventList().size() - 1).addVolunteer(
+                model.getVolunteerStorage().getVolunteerList().get(model.getVolunteerStorage()
+                        .getVolunteerList().size() - 1));
+
+        Index validIndex = Index.fromOneBased(model.getFilteredVolunteerList().size());
+        VolunteerListEventCommand command = new VolunteerListEventCommand(validIndex);
+        Volunteer volunteerToList = model.getVolunteerStorage().getVolunteerList().get(validIndex.getZeroBased());
+
+        String expectedMessage = String.format(VolunteerListEventCommand.MESSAGE_SUCCESS,
+                volunteerToList.getName().fullName, volunteerToList.getAssignedEvents().size());
+
+        try {
+            CommandResult commandResult = command.execute(model);
+            assertEquals(commandResult.getFeedbackToUser(), expectedMessage);
+        } catch (Exception e) {
+            assertTrue(false);
+        }
```

Comment:
try catch block in a test case, is that normal lol feel like I've not seen one before

Reply:
that's fine, but should use JUnit assertions `fail(message)` instead of `assertTrue(false)` although they have the same effect since it's more descriptive

Reply:
good idea, now I changed my test cases to use `fail(message` xd

Reply:
OO ok nice will go and change